apps/src/templates/lessonOverview/activities/ActivitySection.jsx

```diff
@@ -72,7 +74,7 @@ export default class ActivitySection extends Component {
           </div>
           {!isProgressionSection && <SafeMarkdown markdown={section.text} />}
           {isProgressionSection && <ProgressionDetails progression={section} />}
-          <div style={styles.tips}>
+          <div style={{...styles.tips, ...(sectionHasTips && {width: '40%'})}}>
```

Comment:
Just curious, what happens if this `width: '40%'` is always included?

Reply:
If you don't have a tip it still allocates that space so the content like the progression does not fill the whole width